### PR TITLE
Collaborative Repository's Main readme File 

### LINKED ISSUE #7: Main Repository Readme File Documentation
Hamid is working on the main readme file on collaborative repository.

### PR DESCRIPTION
**Pull request by hamid4231**
Hello dear team members!
I have created the main readme file for our collaborative repository. Please feel free to take a look and let me know if any adjustments are needed.
Thank you!

Best,
Hamid

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# 🐍 AI (Actual Intellgence)  
+
+Welcome to the **Actual Intellgence (AI) group**!
```
Comment (KarimMakki) [inline README.md]: Can you fix this Typo in "Intellgence" to "Intelligence"?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Here’s a quick introduction to everyone, along with a fun fact about them:  
+
+- **Karim** ⚽  
+  Fun Fact: I probably spend way too much time watching football and basketball!
+
+- **Nico** 🎞️📺  
+  Fun Fact: I love tv shows. My favs are modern family and Brooklyn nine-nine.
+
+- **Marshar** 🍕  
+  Fun Fact: I can die for food, can't for assignment.
```
Comment (KarimMakki) [inline README.md]: For better grammar, it can be changed to: 'I can die for food but not for assignments'.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+## 👥 Meet the Team  
+
+Our group is a blend of creative minds and problem-solving enthusiasts.
+Here’s a quick introduction to everyone, along with a fun fact about them:  
+
+- **Karim** ⚽  
+  Fun Fact: I probably spend way too much time watching football and basketball!
+
+- **Nico** 🎞️📺  
+  Fun Fact: I love tv shows. My favs are modern family and Brooklyn nine-nine.
```
Comment (KarimMakki) [inline README.md]: "tv shows" → "TV shows," and "modern family" → "Modern Family."

**Review by KarimMakki — CHANGES_REQUESTED** [review]
Hello Hamid, can you fix those Typos/Grammar? I've added a comment on each line that need to be changed

**Comment by hamid4231** [comment]
> Hello Hamid, can you fix those Typos/Grammar? I've added a comment on each line that need to be changed

Hello @KarimMakki, I have made some edits to the readme file according to your suggestions and it should be good to go. Please feel free to have a final check before merging it.
Thank you!

**Review by KarimMakki — APPROVED** [review]

**Comment by KarimMakki** [comment]
Thank you so much @hamid4231 , looks good now!

**Pull request merged**